src/lib/marketingCollectionsCategories.ts

```diff
@@ -0,0 +1,70 @@
+export const marketingCollectionsByCategory = {
+  Medium: {
+    slugs: [
```

Comment:
**suggestion**: It would be nice if this key name reflected that this is a selective, curated, and ordered list of marketing collections, identified by slug and not an exhaustive list of all marketing collections matching this category.

That underlying data and associations are owned by Gravity and could change. Maybe something like `curatedCollectionSlugs`? I think the key name should include `Collection` somewhere as these aren't slugs for the category, but for associated marketing collections.

Reply:
I'm a bit confused by this naming suggestion. The current variable name, `marketingCollectionsByCategory`, already reflects the idea of organized marketing collections by category. IMHO renaming this key might make it a bit more confusing and less clear about the what that key actual represents. Does it make sense?

Reply:
What is the goal of this file?

If it's only to represent the curated/sorted list of marketing collections by category, then we could change it's name and maybe it's structure to something that is focused on that purpose.

Like `orderedMarketingCollectionsByCategory.ts`:

```ts
export default {
  "Collect by Price": [
    "art-under-500-dollars",
    "art-under-1000-dollars",
    "art-under-2500-dollars",
  ]
}
```

If it's purpose is to more generally define Marketing Collection Categories, then we can keep a filename to reflect that and store associated metadata in that structure.

Like `marketingCollectionCategories.ts`:

```ts
export default {
  "Collect by Price": {
    id: "Collect by Price",
    title: "Price",
    orderedCollectionSlugs: [
      "art-under-500-dollars",
      "art-under-1000-dollars",
      "art-under-2500-dollars",
    ]
  }
}
```